bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2ThingHandler.java

```diff
@@ -631,12 +631,37 @@ public void initialize() {
         }
     }
 
+    /**
+     * Update the channel state depending on new resources sent from the bridge.
+     *
+     * @param resources a collection of Resource objects containing the new state.
+     */
+    public void onResources(Collection<Resource> resources) {
+        boolean sceneActivated = false;
+        for (Resource resource : resources) {
+            boolean skipUpdate = sceneActivated && ResourceType.SCENE == resource.getType()
```

Comment:
.. and yes, before you say so, I do admit that my prior PR was not handling smart scenes either ..

Reply:
> OK, if you really want to handle this _perfectly_ then you need to process SMART_SCENE transitions as well. A room or a zone can have either a scene active, a smart scene active, or an inactive state. Therefore you need to handle transitions between all of those states.

I think I only want to handle that if I can reproduce that, or receive payloads for verifying such issue. Using the app I found one scene which is actually a "smart scene": natural light. But it seems it will actually just transition through some "normal" scenes, so this already works. Do you know a way to create a smart scene that will appear in the payload as active/inactive?

Reply:
> Do you know a way to create a smart scene that will appear in the payload as active/inactive?

No.

Reply:
> I think I only want to handle that if I can reproduce that

Perhaps you can understand why @maniac103 thinks that I respond too strongly. If any suggestion is your idea, it is always perfect, no matter how much work is involved for the other party. But only if it is your idea. Whereas if I have an opinion you always say it is too difficult for you to implement, and always rule it out. Perhaps a little more flexibility, please.

Reply:
In response to:
> If any suggestion is your idea, it is always perfect, no matter how much work is involved for the other party. But only if it is your idea. Whereas if I have an opinion you always say it is too difficult for you to implement, and always rule it out. Perhaps a little more flexibility, please.

and: https://github.com/openhab/openhab-addons/pull/16001#issuecomment-1847574483

Sincerely, thanks for giving me this feedback, which will allow me to reflect and self-improve. My communication skills are definitely lacking.

This is no excuse, but just to give you some explanation. Besides the mandatory review feedback about openHAB compliance, conventions, guidelines etc. I try to give individualized reviews based on multiple factors. As an example, when reviewing a high quality contribution, I might suggest improvements I probably would not have suggested otherwise. Contributor history is also a parameter - the better I know a contributor, the better I can help by knowing how much details I should go into. Some appreciate even small comments that will give a slightly better end result, others prefer a faster merge and can get frustrated by too many comments.

I have gotten the impression that you are very detail-oriented, for example based on your review of my contributions and our discussions in general. I might therefore have made some false conclusions about your preferences. I'm sorry for that, and I'll try to recalibrate.

That said, I will also give you some personal feedback. For a long time I have felt hostility from you, and at times just plain rudeness. I'm sensitive to that, so it takes energy and motivation from me. If you have felt ignored, that might have been a consequence of me simply needing to take pauses from that, i.e. reduce interactions or "throttle down" if you will. I have also already tried to put much more consideration into how I phrase things to you, to not upset you. Sometimes it slips, especially if you have really pissed me off, to be frank. It should not, but I am human. Sorry.

Going forward, besides the recalibration I will make, on short term I can try my best to stay away as contributor and reviewer of the bindings for which you are code maintainer. That should reduce tensions, and I also need that for my mental health. I can't promise for how long, since I also enjoy working on improving these bindings. After all, this was my original motivation for becoming a contributor. So it will be a balance act.

Reply:
@jlaur many thanks for this apology; it seems that we may both be 'too detail oriented', and/or have mutually 'p**d each other off'; I therefore offer you also my apology (albeit not as eloquent as above).

My question now is how to proceed? Shall we continue with this PR? Or shall I adapt mine? I think the solution will ultimately be a combination of your code and mine. I have offered some code code below, so either you can take it over; or alternatively I can take over your code into my PR?